xarray/core/common.py

```diff
@@ -613,6 +616,12 @@ def rolling(self, dim: Optional[Mapping[Hashable, int]] = None,
         return self._rolling_cls(self, dim, min_periods=min_periods,
                                  center=center)
 
+    def rolling_exp(self, dim=None, window_type='span', **dim_kwargs):
```

Comment:
Can you add a docstring here? Something short pointing to `RollingExp` for details would be fine, but we should have at least something since this is where users will look first.